gatsby-node.js

```diff
@@ -40,8 +40,8 @@ const createTagPages = (createPage, edges) => {
     });
 };
 
-exports.createPages = ({ boundActionCreators, graphql }) => {
-  const { createPage } = boundActionCreators;
+exports.createPages = ({ actions, graphql }) => {
+  const { createPage } = actions;
```

Comment:
`boundActionCreators` are now `actions`

_Really_ nice thing is that Gatsby CLI warned of deprecated/changed key names.